readme.md

```diff
@@ -102,17 +102,17 @@ Start the spinner. Returns the instance.
 
 Stop and clear the spinner. Returns the instance.
 
-### .succeed()
+### .succeed([text])
 
-Stop the spinner, change it to a green `✔` and persist the `text`. Returns the instance. See the GIF below.
+Stop the spinner, change it to a green `✔` and persist `text` (or `.text` if `text` is not provided). Returns the instance. See the GIF below.
 
-### .fail()
+### .fail([text])
 
-Stop the spinner, change it to a red `✖` and persist the `text`. Returns the instance. See the GIF below.
+Stop the spinner, change it to a red `✖` and persist `text` (or `.text` if `text` is  not provided). Returns the instance. See the GIF below.
 
-### .stopAndPersist([symbol])
+### .stopAndPersist([symbol[, text]])
```

Comment:
Wondering if it's a better idea to use an options object here. Both `symbol` and `text` are optional and this way it's not possible to change the `text` without providing the `symbol`.